src/topology/homotopy/fundamental_groupoid.lean

```diff
@@ -291,3 +292,39 @@ instance : category_theory.groupoid (fundamental_groupoid X) :=
   comp_inv' := λ x y f, quotient.induction_on f
     (λ a, show ⟦a.trans a.symm⟧ = ⟦path.refl x⟧,
           from quotient.sound ⟨(path.homotopy.refl_trans_symm a).symm⟩) }
+
+lemma comp_eq (x y z : fundamental_groupoid X) (p : x ⟶ y) (q : y ⟶ z) :
+  p ≫ q = quotient.map₂ path.trans
+    (λ (p₀ : path x y) p₁ hp q₀ q₁ hq, path.homotopic.hcomp hp hq) p q := rfl
+
+/--
+The functor sending a topological space `X` to its fundamental groupoid.
+-/
+def π₁ : Top ⥤ category_theory.Groupoid :=
```

Comment:
```suggestion
def fundamental_groupoid_functor : Top ⥤ category_theory.Groupoid :=
```
It's fine to have local notation `π₁` for this. But I imagine that we might want to be able to use that notation for other things as well (e.g. fundamental group of a Kan complex, etc).